app/src/main/java/org/wikipedia/WikipediaApp.java

```diff
@@ -401,6 +401,7 @@ private void updateCrashReportProps() {
             if (appLanguageState != null) {
                 putCrashReportProperty("app_primary_language", appLanguageState.getAppLanguageCode());
                 putCrashReportProperty("app_languages", appLanguageState.getAppLanguageCodes().toString());
+                putCrashReportProperty("app_install_id", getAppInstallID());
```

Comment:
This is a pretty good idea; it would let us look through eventlogging and see in which tables this ID pops up.